src/h_vialib/__init__.py

```diff
@@ -1,4 +1,13 @@
-"""Library functions for Via related products."""
+"""Library functions for Via related products.
+
+isort:skip_file
+
+isort disabled to keep manual ordering to avoid circular dependencies
```

Comment:
Ouch! What's the circular dependency and can we break it another way?

Reply:
Here's the fix:

```diff
diff --git a/src/h_vialib/__init__.py b/src/h_vialib/__init__.py
index 59e7eb5..1d22957 100644
--- a/src/h_vialib/__init__.py
+++ b/src/h_vialib/__init__.py
@@ -1,13 +1,5 @@
-"""Library functions for Via related products.
-
-isort:skip_file
-
-isort disabled to keep manual ordering to avoid circular dependencies
-"""
+"""Library functions for Via related products."""
 
 
+from h_vialib.client import ViaClient, ViaDoc
 from h_vialib.configuration import Configuration
-from h_vialib.client import (
-    ViaClient,
-    ViaDoc,
-)
diff --git a/src/h_vialib/secure/token.py b/src/h_vialib/secure/token.py
index ca103a1..d183cb2 100644
--- a/src/h_vialib/secure/token.py
+++ b/src/h_vialib/secure/token.py
@@ -5,7 +5,7 @@ from urllib.parse import urlparse, urlunparse
 import jwt
 from jwt import DecodeError, ExpiredSignatureError, InvalidSignatureError
 
-from h_vialib import Configuration
+from h_vialib.configuration import Configuration
 from h_vialib.exceptions import InvalidToken, MissingToken
 from h_vialib.secure.expiry import as_expires, quantized_expiry
 
```

I think generally you want _internal_ code to import from the internal locations (e.g. `from h_vialib.configuration import Configuration`) and the public interface (`from h_vialib import Configuration`) is only for external code to import from. To avoid circular dependencies.